### ISSUE TITLE
FormatSheet display behind OS keyboard on IOS 17

### ISSUE THREAD

**Issue by minhtien1403** [danielsaidi/RichTextKit#85]
I update to Xcode15 - IOS 17 and this bug happen, the format text sheet is behind the os keyboard
<img src="https://github.com/danielsaidi/RichTextKit/assets/54258059/8738bdb1-7208-40df-8fa6-30cfd6d9ee34" width=250>

**Comment by danielsaidi** [comment]
Oh snap! This doesn't happen in iOS 16, right? 

Are you using the latest release? I changed the format toolbar to use a medium size and to not dim the background.

So it may not be an iOS 17 thing, but rather that I overlooked this scenario.

**Comment by minhtien1403** [comment]
It's oke when I test with 16.4 but it maybe still happen on IOS > 16.4
I'm using Xcode 15.0.1 now

**Comment by minhtien1403** [comment]
```
func prefersMediumSize() -> some View {
        if #available(iOS 16.4, *) {
            self.presentationDetents([.medium])
                .presentationBackgroundInteraction(
                    .enabled(upThrough: .medium)
                )
        } else if #available(iOS 16, *) {
            self.presentationDetents([.medium])
        } else {
            self
        }
    }
```
I found that everything is working oke when I just use `self.presentationDetents([.medium])` for IOS higher than 16.4. I wonder why you have to this code 
```
.presentationBackgroundInteraction(
                    .enabled(upThrough: .medium)
                )
```

**Comment by hardeyem** [comment]
Hello @minhtien1403 are you able to solve this problem. I am trying to use the library on IOS and i am having the same issue. IOS 16.6

**Comment by minhtien1403** [comment]
@hardeyem yeah, I have explain my solution about and its work, you have to edit source code of this library or waiting for a hotfix from  @danielsaidi 
https://developer.apple.com/documentation/xcode/editing-a-package-dependency-as-a-local-package

**Comment by hardeyem** [comment]
Thank you @minhtien1403

**Comment by danielsaidi** [comment]
I'll revert this change.

**Comment by danielsaidi** [comment]
Ok, I have reverted the 0.9 addition that kept the background active when the format sheet was presented, since this caused the sheet to display behind the keyboard.

I would love for this behave like in Notes, where the format sheet dismisses the keyboard but keeps the text field editable. When I tried this and dismissed the keyboard when presenting the sheet, the text field loses focus.

Anyhow, we can improve this over time. For now, reverting this will restore the previous behavior. 👍
